frontend/src/components/CategorySelect.tsx

```diff
@@ -112,11 +120,27 @@ const CategorySelect: FC<ICategorySelectProps> = ({
           '& .MuiSelect-icon': { color: '#f2f2f2' },
         }}
       >
-        {categories.map((category, i) => (
-          <MenuItem key={i} value={category.id} sx={{ fontSize: '0.85em', fontWeight: 600 }}>
-            {category.name}
-          </MenuItem>
-        ))}
+        {categories.map((category, i) => {
+          return (
+            <MenuItem key={i} value={category.id} sx={{ fontSize: '0.85em', fontWeight: 600 }}>
+              {category.name}
+            </MenuItem>
+          );
+        })}
+        <MenuItem
+          sx={{
+            paddingRight: '5rem',
+            color: '#adb2b8',
+            fontSize: '0.85em',
+            fontWeight: 600,
+          }}
+          key={NaN}
+        >
+          <ListItemIcon sx={{ margin: 0 }}>
+            <AddIcon fontSize="small" />
+          </ListItemIcon>
+          {t(StringBank.ADD_NEW_CATEGORY)}
+        </MenuItem>
```

Comment:
```suggestion
        {categories.map((category, i) => (
          <MenuItem key={i} value={category.id} sx={{ fontSize: '0.85em', fontWeight: 600 }}>
            {category.name}
          </MenuItem>
        ))}
        <MenuItem>
          <AddIcon fontSize="small" htmlColor="#adb2b8" />
          <Typography variant="body2" color="#adb2b8" paddingX={1}>
            {t(StringBank.ADD_NEW_CATEGORY)}
          </Typography>
        </MenuItem>
```

Reply:
Short explanation for the suggestion:
1. we don't need to use a `key` prop outside a loop
2. We already have a typography variant that implements font size of 0.875rem (or 14px), and normal weight, which is body2. So I used it and added some coloring and padding according to the zeplin.